src/klicky.ts

```diff
@@ -1,3 +1,54 @@
 export const Klicky = () => {
   console.log("klickly ran!");
 };
+
+const outer = document.querySelector('.outer');
+
+const capture = {
+  capture : true
+};
+
+const noneCapture = {
+  capture : false 
+};
+
+const once = {
+  once : false 
+};
+
+outer?.addEventListener('click', onceHandler, once);
+
+function onceHandler(event)
+{
+  console.log("You have clicked the page");
+  write();
+  var date = new Date();
+  dateTime(date);
+}
+
+function write()
```

Comment:
Going along with the ES6 comment above, can you convert this into an arrow function? You may have to move this function towards the top as ES6 functions don't get hoisted. This means that if you want to use a function, it must be declared before instantiation

Also, could you make the name of this function more descriptive?

Reply:
Converted and renamed function